conbench/templates/index.html

```diff
@@ -54,28 +54,43 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
                      </td>
 
                      <td>
+                         <div class="table-entry">
                        {% if run.commit.author_avatar %}
-                         <image src="{{ run.commit.author_avatar }}" height="30" style="border-radius: 50%;">&nbsp;
+                         <img src="{{ run.commit.author_avatar }}" height="30" style="border-radius: 50%;">&nbsp;
                        {% endif %}
                        {{ run.commit.author_name }}
+                         </div>
                      </td>
 
-                     <td><a href="{{ run.commit.repository }}">{{ run.commit.display_repository }}</a></td>
+                    <td>
+                         <div class="table-entry">
+                             <a href="{{ run.commit.repository }}">{{ run.commit.display_repository }}</a>
+                         </div>
+                    </td>
 
-                     {% if run.commit.display_message %}
-                       <td><a href="{{ run.commit.url }}">{{ run.commit.sha[:7] }} {{ run.commit.display_message }}</a></td>
-                     {% else %}
-                       <td><a href="{{ run.commit.url }}">{{ run.commit.sha }}</a></td>
-                     {% endif %}
+                    <td>
+                         <div class="table-entry">{{ run.hardware.name }}</div>
+                    </td>
 
-                     <td>{{ run.commit.sha }}</td>
-                     <td><div class="ellipsis-175">{{ run.hardware.name }}</div></td>
+                    <td>
+                         <div class="table-entry"><a href="{{ run.commit.url }}">{{ run.commit.sha[:7] }}</a></div>
+                    </td>
 
-                     {% if run.reason %}
-                       <td><div class="ellipsis-175">{{ run.reason }}</div></td>
-                     {% else %}
-                       <td></td>
-                     {% endif %}
+                    <td>
+                         {% if run.commit.display_message %}
+                            <div class="table-entry">{{ run.commit.display_message }}</div>
+                         {% else %}
+                            <div class="table-entry">{{ run.commit.sha }}</div>
```

Comment:
Maybe if there is no display message this should be just a blank string instead of the 40-character sha?

Reply:
I was thinking on similar lines, not sure if the users might be expecting the hash instead?

Reply:
Blank made more sense (to me) I added the change. We can reverse if necessary